nbs/99_manuscript/intro/py/05-relationships_analysis.py

```diff
@@ -0,0 +1,313 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: all,-execution,-papermill,-trusted
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.5
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown] tags=[]
+# # Description
+
+# %% [markdown] tags=[]
+# It generates datasets showing different relationship types to compare correlation coefficients.
```

Comment:
This description was a little confusing to me. Based on what it says, I would assume that this notebook was generating datasets to compare the different correlation metrics, which it does. But you also compare the different metrics here as well. Maybe also describe what you expect to see..?

Reply:
Yes, perfect, added!